ts/src/header-validator/source.test.ts

```diff
@@ -1314,7 +1317,10 @@ const testCases: TestCase[] = [
   },
   {
     name: 'trigger-state-cardinality-invalid',
-    json: `{"destination": "https://a.test"}`,
+    json: `{
+      "destination": "https://a.test",
+      "max_event_states": 2
```

Comment:
otherwise it will fail with a different error `max event states > max trigger-state cardinality`, as the default `max event states` is set to 3, and `max trigger-state cardinality` is 2 here. 

Didn't want to create another event source with more than 3 trigger states, so I decided to explicitly set max event states to be smaller.